### ISSUE TITLE
Real yet high quality screenshot

### ISSUE THREAD

**Issue by installgen2** [installgen2/tox.chat#9]
Also, only serve high-res screenshot to devices that need it.

**Comment by prayashm** [comment]
I want to work on this issue, how should I proceed?

**Comment by ProMcTagonist** [comment]
1) Work out a cute script for a conversation that shows off file transfers, actions, and emoji.
2) Act out the conversation with someone in a tox instance with a few friends with good names and avatars
3) Take screenshots of this on as many platforms as you have access to

Anything else, @installgen2?

**Comment by Zer0-One** [comment]
@prayashm what @ProMcTagonist suggested will do. The important thing is that we have a set of high-quality images of each client (on each platform) that show off their features. If you can do short video-captures, that might also be of some use.

If you decide to work on this, please make a serious effort to keep the text and images on the screen professional (something that everyone involved in this project seems to have a hard time doing).

**Comment by installgen2** [comment]
What's more important right now than screenshots of different platforms is resolution, some qTox pics on a Retina MacBook would be nice, or preferably a freedom-respecting way to take high-DPI screenshots.

**Comment by anoadragon453** [comment]
I've got a Macbook but not a good selection of contacts to make a nice screenshot. People can add me here if to fill it up: https://toxme.io/u/anoa

**Comment by Zer0-One** [comment]
@tux3 @grayhatter @JFreegman @wiiaam Please provide a few screenshots of your respective clients that you'd like displayed on the website.

**Comment by JFreegman** [comment]
http://i.imgur.com/ofpfBNU.png

**Comment by Zer0-One** [comment]
@JFreegman Not really, no. Ideally you'd have a few different images that all show off different features.

**Comment by SkyzohKey** [comment]
@JFreegman I think we should make a "discussion-type" that all the screenshots should follow, it would permit to users not to be loosed while comparing clients.

**Comment by GrayHatter** [comment]
You can pull the images from the uTox repo

**Comment by rugk** [comment]
> I think we should make a "discussion-type" that all the screenshots should follow, it would permit to users not to be loosed while comparing clients.

Ideally would be to have a standardized qTox example `.tox` file (and chat history and so on), which clients can load, so they always display the same content (aka chat, users, ...). 
This way visitors of your website can really *only* focus on the design when comparing the screenshots and the content would always be the same.

I think this would be awesome. Of course the template tox must show all features of tox.
